app/_how-tos/add-rate-limiting-for-a-consumer-with-kong-gateway.md

```diff
@@ -107,8 +106,15 @@ append_to_existing_section: true
 ## 5. Validate
 
 You can run the following command to test the rate limiting as the consumer:
+
 ```bash
 for _ in {1..6}; do curl -i http://localhost:8000/example-route -H 'apikey:example_key'; echo; done
 ```
+{: data-deployment-topology="on-prem" }
+
+```bash
+for _ in {1..6}; do curl -i http://{host}/example-route -H 'apikey:example_key'; echo; done
```

Comment:
@Guaris What do you think of the `{host}` here? Since we don't know their host for the route. Or at least I thought this was the host for the route...